package.json

```diff
@@ -15,7 +15,7 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">=16.0.0"
+    "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
```

Comment:
Node 21 is EOL. Node 18 is EOL.

Reply:
I want to match the CLI so I don't move faster than it.

 "engines": {
    "node": "18.20.7 || ^20.18.3 || >=22.14.0"
  },

Reply:
```suggestion
    "node": "18.20.7 || ^20.18.3 || >=22.14.0"
```